timescaledb/tutorials/promscale/promscale-benefits.md

```diff
@@ -1,14 +1,14 @@
-# 1. Why use Promscale & TimescaleDB to store Prometheus metrics [](why-promscale)
+# Why use Promscale & TimescaleDB to store Prometheus metrics [](why-promscale)
 
 This section details five reasons to use Timescale and Promscale to store Prometheus metrics:
 
-1. Long term data storage
-2. Operational ease
-3. Queries in SQL and PromQL
-4. Per Metric Retention
-5. Ability to push data from custom applications
+1. [Long term data storage](#long-term-data-storage)
+2. [Operational ease](#operational-ease)
+3. [Queries in SQL and PromQL](#queries-in-sql-and-promql)
+4. [Per Metric Retention](#per-metric-retention)
+5. [Ability to push data from custom applications](#ability-to-push-data-from-custom-applications)
```

Comment:
With Jacob's fixes from a few weeks ago, this page _should_ use the defined anchor links and not the titles, so I think these would be the correct in-page anchor's to use.

```suggestion
1. [Long term data storage](#data-storage)
2. [Operational ease](#operational-ease)
3. [Queries in SQL and PromQL](#queries)
4. [Per Metric Retention](#per-metric)
5. [Ability to push data from custom applications](#custom-apps)
```